lib/screens/dashboard.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:bytebank/components/features/lista.dart';
+import 'package:bytebank/components/quadro.dart';
+import 'package:bytebank/screens/lista_transferencias.dart';
+import 'package:flutter/material.dart';
+
+import 'lista_contatos.dart';
+
+const _imagemDashboard = 'assets/logo-bytebank.png';
+
+const _tituloAppBar = 'Dashboard';
+
+class Dashboard extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(_tituloAppBar),
+      ),
+      body: Container(
+        color: Colors.white,
```

Comment:
Scaffolds also have a property called backgroundColor, if that's all this Container is for